cli/create_db.sh

```diff
@@ -0,0 +1,10 @@
+#1/bin/bash
+
+# "who am i" responds correctly even with sudo
+NEST_USER=$(who am i | awk '{print $1}')
+
+NAME="${NEST_USER}_$1"
+
+sudo -u postgres createdb -O $NEST_USER $NAME
```

Comment:
This may cause unexpected behavior with database names that contain spaces.
```
h@nest:~$ export e="a b"
h@nest:~$ q() {
> echo $1
> }
h@nest:~$ q $e
a
h@nest:~$ 
```
Also, rather than running this script as root and dropping privileges, why not just only authorize users to run it as `postgres` (or maybe even a user with a postgres role that's only able to create databases, rather than having full database superuser)?